### PR TITLE
ibacm: Add comments explaining mc join bits

### PR DESCRIPTION
**Pull request by Hakon-Bugge**
Signed-off-by: Håkon Bugge <haakon.bugge@oracle.com>

### CODE REVIEW

File: ibacm/prov/acmp/src/acmp.c [hunk 1]
```diff
@@ -1380,15 +1380,15 @@ static void acmp_init_join(struct ib_sa_mad *mad, union ibv_gid *port_gid,
 		IB_COMP_MASK_MC_SCOPE | IB_COMP_MASK_MC_JOIN_STATE;
 
 	mc_rec = (struct ib_mc_member_rec *) mad->data;
-	acmp_format_mgid(&mc_rec->mgid, pkey | 0x8000, tos, rate, mtu);
+	acmp_format_mgid(&mc_rec->mgid, pkey | 0x8000, tos, rate, mtu); /* Full partition mmbr (0x8000) */
```
Comment (weiny2) [inline ibacm/prov/acmp/src/acmp.c]: Would it be worth adding #defines for values like this?  I'm ok with the patch as I'm familiar with the meanings of these bits.  But if it was troublesome enough for you to want to add a comment here maybe we should have something like:

#define RDMA_PKEY_FULL_MEMBER 0x8000
#define RDMA_MTU_EXACT 0x80
#define RDMA_RATE_EXACT 0x80

It's been a while so I apologize if they have been defined somewhere like in the umad headers.

Comment (Hakon-Bugge) [reply ibacm/prov/acmp/src/acmp.c]: The most important, agnostic to the use of defines or comments, is to use terms that can be found in the relevant collaterals, i.e. IBTA spec in this setting.
I used some hours this morning to try to understand the code, and said to myself, if have have done this, just let me document it for others as well.

Comment (jgunthorpe) [reply ibacm/prov/acmp/src/acmp.c]: I think Ira is right, this should be done by adding new #defines, not by sprinkling more magic constant values and comments  around

Comment (Hakon-Bugge) [reply ibacm/prov/acmp/src/acmp.c]: @jgunthorpe , @weiny2 I'll work on this after I have gotten the functional commits in.

Comment (Hakon-Bugge) [reply ibacm/prov/acmp/src/acmp.c]: I am deleting this PR, replaced by PR #427

**Comment by Hakon-Bugge** [comment]
I am deleting this PR, replaced by PR #427

**Pull request closed without merging**